traffic_router/core/src/main/conf/log4j2.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!--
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+ -->
+<Configuration>
+    <Appenders>
+        <Console name="stdout" target="SYSTEM_OUT">
+            <PatternLayout pattern="%m%n" />
+        </Console>
+        <RollingFile name="traffic_router_access"
+                     fileName="/opt/traffic_router/var/log/access.log"
+                     filePattern="/opt/traffic_router/var/log/access-%d{yyyyMMdd}.log" >
+            <PatternLayout pattern="%m%n" />
+            <Policies>
+                <SizeBasedTriggeringPolicy size="200MB" />
+            </Policies>
+            <DefaultRolloverStrategy max="10"/>
+        </RollingFile>
+        <RollingFile name="traffic_router"
+                     fileName="/opt/traffic_router/var/log/traffic_router.log"
+                     filePattern="/opt/traffic_router/var/log/traffic_router-%d{yyyyMMdd}.log" >
+            <Policies>
+                <SizeBasedTriggeringPolicy size="100MB" />
+            </Policies>
+        </RollingFile>
+    </Appenders>
+    <Loggers>
+        <Logger name="org.apache.traffic_control.traffic_router" level="INFO" />
+        <Logger name="org.apache.traffic_control.traffic_router.core.access" level="INFO" >
+            <AppenderRef ref="ACCESS"/>
+        </Logger>
+        <Root level="WARN" additivity="false">
+            <AppenderRef ref="console" />
```

Comment:
I don't see an Appender defined above that is named `"console"`.

Reply:
![image](https://user-images.githubusercontent.com/22248619/146092707-f0188156-e5d9-4df4-a5fe-ecfdf3d5dbde.png)